src/main/java/SYSC6/Project/AdminController.java

```diff
@@ -1,35 +1,29 @@
 package SYSC6.Project;
 
 import SYSC6.Project.user.User;
-import SYSC6.Project.user.UserUtil;
-import org.json.simple.JSONObject;
-import org.json.simple.parser.JSONParser;
-import org.json.simple.parser.ParseException;
+import SYSC6.Project.user.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
-
 @Controller
 public class AdminController {
 
     private Long id = 0L;
 
-    @GetMapping("/admin_portal")
-    public String greeting_admin(@RequestParam(name="name", required=false, defaultValue="World") String name_place, Model model) {
-        User user = getUser(id);
-        name_place = user.getUsername();
-        model.addAttribute("name", "Admin");
-        user.setRole(RoleType.ADMIN);
-        model.addAttribute("role", user.getRole().toString());
-        return "admin_portal";
-    }
+    @Autowired
+    UserRepository userRepository;
+//
+//    @GetMapping("/admin_portal")
+//    public String greeting_admin(@RequestParam(name="name", required=false, defaultValue="World") String name_place, Model model) {
+//        User user = userRepository.findByUsername("admin");
+//        name_place = user.getUsername();
+//        model.addAttribute("name", "Admin");
+//        user.setRole(RoleType.ADMIN);
+//        model.addAttribute("role", user.getRole().toString());
+//        return "admin_portal";
+//    }
```

Comment:
Please remove the comments from this file if not necessary

Reply:
Remove these comments as well since they are not being used

Reply:
This file has been deleted already